comps/third_parties/vllm/src/build_docker_vllm.sh

```diff
@@ -35,19 +35,14 @@ fi
 
 # Build the docker image for vLLM based on the hardware mode
 if [ "$hw_mode" = "hpu" ]; then
-    git clone https://github.com/HabanaAI/vllm-fork.git && cd vllm-fork
-    VLLM_VER=v0.6.6.post1+Gaudi-1.20.0
-    echo "Check out vLLM tag ${VLLM_VER}"
-    git checkout ${VLLM_VER} &> /dev/null
+    source $(git rev-parse --show-toplevel)/.github/env/_vllm_versions.sh
```

Comment:
This is nice as all scripts can use the same line regardless of where the script is (easy to grep etc), and it continues working even when the script is moved.

However, this does not work when user gets sources through some other means that Git, e.g. by downloading them from the releases page: https://github.com/opea-project/GenAIComps/releases

I don't think supporting building these images without Git checkout to be important enough use-case, to change this, but it's possible that somebody will file a bug about it.

If you're worried about that, you need to replace git invocations with hard-coded path. :-/